src/api/controllers/__init__.py

```diff
@@ -0,0 +1 @@
+"""Controllers Init."""
```

Comment:
Our standard blurb looks like this:
```suggestion
"""The controllers subpackage."""
```